Batteries/Data/Array/Lemmas.lean

```diff
@@ -31,14 +33,14 @@ theorem idxOf?_toList [BEq α] {a : α} {l : Array α} :
 @[deprecated (since := "2025-02-06")] alias eraseP_toArray := List.eraseP_toArray
 @[deprecated (since := "2025-02-06")] alias erase_toArray := List.erase_toArray
 
-@[simp] theorem toList_erase [BEq α] (l : Array α) (a : α) :
+@[simp, grind =] theorem toList_erase [BEq α] (l : Array α) (a : α) :
     (l.erase a).toList = l.toList.erase a := by
   rcases l with ⟨l⟩
   simp
 
-@[simp] theorem size_eraseIdxIfInBounds (a : Array α) (i : Nat) :
+@[simp, grind =] theorem size_eraseIdxIfInBounds (a : Array α) (i : Nat) :
```

Comment:
I will add a grind annotation in Lean that allows grind to unfold `eraseIdxIfInBounds`, after which this won't be necessary.

Reply:
https://github.com/leanprover/lean4/pull/9120